etna/whatson/models.py

```diff
@@ -290,6 +290,15 @@ def filter_form_data(self, filter_form_cleaned_data):
 
         return events
 
+    # we need a separate query to check if the featured event is filtered out or not
+    def should_show_featured_event(self, filtered_events, featured_event):
```

Comment:
```suggestion
    def should_show_featured_event(self, filtered_events):
```

Issue: unused argument